core/context.go

```diff
@@ -113,13 +110,9 @@ func (c *Context) WithFrame(f frame.Frame) {
 	c.Frame = f
 }
 
-// CloseWithError close dataStream in se error,
-// It tells controlStream which dataStream should be closed and close dataStream with
-// returning error message to client side stream.
-//
-// TODO: ycode was not transmitted.
-func (c *Context) CloseWithError(ycode yerr.ErrorCode, errString string) {
-	c.Logger.Warn("data stream closed", "err_code", ycode.String(), "error", errString)
+// CloseWithError close dataStream with an error string.
+func (c *Context) CloseWithError(errString string) {
```

Comment:
Remove `ycode` parameter as it is not transmitted in the `CloseWithError` function.